clients-src/modules/ROOT/pages/console.adoc

```diff
@@ -25,6 +25,12 @@ It has unique set of functions and abilities to interact with a TypeDB database:
 
 TypeDB Console is distributed with TypeDB server and does not need an installation.
 
+[IMPORTANT]
+====
+We recommend using the TypeDB Console from the TypeDB server files directory.
```

Comment:
"We recommend using the version of TypeDB Console that's packaged with the TypeDB server."